src/tools/parser/KomposTraceParser.java

```diff
@@ -0,0 +1,327 @@
+package tools.parser;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.channels.FileChannel;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+
+import som.Output;
+import som.vm.VmSettings;
+import tools.TraceData;
+import tools.debugger.entities.Marker;
+
+
+public class KomposTraceParser {
+
+  private static Set<Long> errorStrackTrace;
+
+  private enum TraceRecords {
+    ActivityCreation(11 + TraceData.SOURCE_SECTION_SIZE),
```

Comment:
I think, these magic constants are still defined elsewhere.
Examples are:
 - https://github.com/smarr/SOMns/blob/385926d94f7dd3037e51e713316fd8b8b39b501b/src/tools/debugger/entities/ActivityType.java#L56-L57
 - https://github.com/smarr/SOMns/blob/385926d94f7dd3037e51e713316fd8b8b39b501b/src/tools/debugger/entities/PassiveEntityType.java#L36-L37

I would really prefer if these magic numbers are defined exactly once, and that we then simply refer to those where needed.
Otherwise, we are bound to run into trouble when we start changing things.